types/src/unit_tests/canonical_serialization_examples.rs

```diff
@@ -1,264 +0,0 @@
-// Copyright (c) Aptos
-// SPDX-License-Identifier: Apache-2.0
-
-//! These tests verify the behavior of BCS against some known test vectors with various types.
-
-use crate::{
-    access_path::AccessPath,
-    account_address::AccountAddress,
-    account_config::XUS_NAME,
-    chain_id::ChainId,
-    transaction::{
-        ChangeSet, RawTransaction, Script, TransactionArgument, TransactionPayload, WriteSetPayload,
-    },
-    write_set::{WriteOp, WriteSet, WriteSetMut},
-};
-use bcs::to_bytes;
-
-#[test]
-fn test_access_path_canonical_serialization_example() {
-    let account_address = AccountAddress::new([
```

Comment:
we probably should fix it instead of deleting

Reply:
fixed.